etc/fpgas.json

```diff
@@ -3,7 +3,7 @@
       "vc707": {
         "id": "10ee:7021",
         "slot": "0000:88:00.0",
-        "do_reset": true,
+        "do_reset": false,
```

Comment:
Just wondering: is this intended? Yesterday you mentioned that a reset will still be performed?